src/components/Toggle/index.js

```diff
@@ -0,0 +1,38 @@
+import './index.scss';
+import { Component, createIDFactory } from 'pet-dex-utilities';
+
+const generateID = createIDFactory('toggle');
+
+const events = ['toggle'];
+
+const html = `
+  <div class="toggle-container">
+    <input type="checkbox" id="toggle-input" class="toggle-container__input" data-select="toggle-input">
+    <label for="toggle-input" class="toggle-container__label" data-select="toggle-label"></label>
+  </div>
+`;
+
+export default function Toggle(checked) {
```

Comment:
Prefira receber um objeto com todas as props e tente sempre receber valores default

```suggestion
export default function Toggle({ checked = false } = {}) {
```

Reply:
Esse tipo de objeto é novo para mim, poderia explicar melhor?